consensus/pacemaker/pacemaker.go

```diff
@@ -8,22 +8,21 @@ import (
 
 	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
 	typesCons "github.com/pokt-network/pocket/consensus/types"
+	"github.com/pokt-network/pocket/shared/codec"
 	"github.com/pokt-network/pocket/shared/modules"
+	"google.golang.org/protobuf/types/known/anypb"
 )
 
 const (
 	pacemakerModuleName = "pacemaker"
+	NewRound            = typesCons.HotstuffStep_HOTSTUFF_STEP_NEWROUND
```

Comment:
Aren't these constants already defined elsewhere?

Reply:
they are defined in consensus module, but we can't import consensus module to use this constants due to import cycle. Therefore, they are defined again for Pacemaker module.